grafana-tikv-dashboard.md

```diff
@@ -484,6 +484,47 @@ This section provides a detailed description of these key metrics on the **TiKV-
 - Initial Scanning Trigger Reason: The reason for triggering incremental scanning
 - Region Checkpoint Key Putting: The number of checkpoint operations logged to the PD
 
+### Snapshot restore 
+
+- Import CPU Utilization: CPU utilization aggregated by sst importer.
+- Import Thread Count: number of threads used by sst importer.
+- Import Errors: error counts during sst import.
+- Import RPC Duration: time spent for various RPC calls in sst importer.
+- Import RPC Ops: number of total RPC calls in sst importer.
+- Import RPC Count: number of inflight RPC calls in sst importer.
+- Import Write/Download RPC Duration: RPC time for write/download in sst importer.
+- Import Wait Duration: time spent on downloading task waiting in queue for execution.
+- Import Read SST Duration: time spent on reading from external storage of a file and download it to TiKV.
+- Import Rewrite SST Duration: time spent on rewriting SST based on rewrite rules. 
+- Import Ingest RPC Duration: time spent on sending ingest response in RPC call.
```

Comment:
```suggestion
- Import Ingest RPC Duration: time spent on handing ingest response in RPC call.
```

It is server-side.

Reply:
I see, just read the code again, you are right, the timer is started before the handling logic, I thought it was the time of sending response from server back to client.
The name of RPC duration is confusing to me actually. Normally it should measure the entire GRPC call end to end.
```
// client side
timer = Time();
server.sendGrpcRequest().await;
record(timer);
```
in our use case it is only measuring part of it on the server side. we should audit all those metrics and rename it properly.